### PR TITLE
Add unit testing for model converters

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*
- Add unit testing for model converters
- decouple code from `helpers.ts`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by akbhatta — APPROVED** [review]

File: kibana-reports/server/routes/utils/converters/__tests__/backendToUi.test.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import {
```
Comment (davidcui-amzn) [inline kibana-reports/server/routes/utils/converters/__tests__/backendToUi.test.ts:1]: missing copyright notice

File: kibana-reports/server/routes/utils/converters/__tests__/uiToBackend.test.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { ReportDefinitionSchemaType } from 'server/model';
```
Comment (davidcui-amzn) [inline kibana-reports/server/routes/utils/converters/__tests__/uiToBackend.test.ts:1]: missing copyright notice

**Review by davidcui-amzn — APPROVED** [review]
LGTM- please address comments before merging

File: kibana-reports/server/routes/utils/converters/__tests__/uiToBackend.test.ts (hunk 2 above)
Comment (zhongnansu) [reply kibana-reports/server/routes/utils/converters/__tests__/uiToBackend.test.ts:1]: will do

**Pull request merged**